test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js

```diff
@@ -1503,22 +1234,47 @@ describe('test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js', () => {
             headObject: async (params) => generateMatchingHeadObjectResponse(serverless, params),
           },
           CloudWatchLogs: {
-            describeSubscriptionFilters: () => {
-              // TODO: Ensure same ARN as lambda on which it is configured, but tweak index for one of the filters
-            },
-            // TODO: Configure stub
-            deleteSubscriptionFilter: null,
+            deleteSubscriptionFilter: deleteStub,
+            describeSubscriptionFilters: sandbox
+              .stub()
+              .onFirstCall()
+              .callsFake(() =>
+                Promise.resolve({
+                  subscriptionFilters: [
+                    {
+                      filterName: 'teststack-Fn1LogsSubscriptionFilterCloudWatchLog1-xxxxx',
+                      destinationArn: `arn:aws:lambda:us-east-1:999999999999:function:${
+                        serverless.service.getFunction('fn1').name
+                      }`,
+                    },
+                  ],
+                })
+              )
+              .onSecondCall()
+              .callsFake(() =>
+                Promise.resolve({
+                  subscriptionFilters: [
+                    {
+                      // someLogGroupeName2 was previously the third event
+                      filterName: 'teststack-Fn1LogsSubscriptionFilterCloudWatchLog3-xxxxx',
+                      destinationArn: `arn:aws:lambda:us-east-1:999999999999:function:${
+                        serverless.service.getFunction('fn1').name
+                      }`,
+                    },
+                  ],
+                })
+              ),
           },
         },
       });
-
-      // TODO: Confirm that stub (for filter in question) was called
+      expect(deleteStub).to.have.been.calledOnceWith({
+        logGroupName: 'someLogGroupName2',
+        filterName: 'teststack-Fn1LogsSubscriptionFilterCloudWatchLog3-xxxxx',
+      });
     });
 
     it('should not attempt to delete and add filter in context of custom partition', async () => {
-      // Replaces:
-      // https://github.com/serverless/serverless/blob/61dd3bde8d17cdd995fdd27259a689d12bee1e42/test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js#L785-L814
-
+      const deleteStub = sandbox.stub();
```

Comment:
If see correctly, this test is simply to confirm that we recognized custom partitions (as `aws-us-gov`), let's update it's title to _"should recognize custom partition"_ (current title is misleading, as it suggests that we simply never delete filters if it's set in custom partition, that's not true)

Reply:
Title changed.